### ISSUE TITLE
Keyboard (WCAG 2.1.1a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1233]
**Request:** The intent of this Success Criterion is to ensure that, wherever possible, content can be operated through a keyboard or keyboard interface (so an alternate keyboard can be used).

**Issues:** 
1. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 

- Grayed out buttons at the bottom of the page should not be focusable. The buttons should be given the attribute `disabled` so that their status is correctly passed on to the screen reader.

2. https://slidewiki.org/user/vivien.zschammer/recommendations
- Sorting of the results is not possible because the buttons provided are not accessible via the keyboard.

![grafik](https://user-images.githubusercontent.com/48785136/74315630-20cde480-4d78-11ea-95ae-8108866ff767.png)

3. https://slidewiki.org/user/vivien.zschammer/stats 

- The tooltips in the coordinate system can be reached with the mouse, but not with the keyboard. This may result in the user losing important information.

**Comment by aoelen** [comment]
1. Unfortunately this is in a dependency, i.e. it is hard to change this. For now, I suggest to postpone this item 

2. For me, it works using the arrow keys to operate this dropdown (arrow up/down). Can you confirm this?

**Comment by VivienDZB** [comment]
Yes, I can confirm that.
